Update sharepoint-site-usage-ww.md (#12334)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/activity-reports/sharepoint-site-usage-ww.md

|Secure link for guest count  |The number of times documents or folders are shared using "specific people" on the site.  |
|Secure link for member count  |The number of times documents or folders are shared using "specific people" on the site.  |
|Root Web Template  |The template used for creating the site.  <br/> NOTE: If you want to filter the data by different site types, then export the data and use the Root Web Template column. |

> Note that you may see differences between the sites listed above and those listed on the [Active sites page](https://go.microsoft.com/fwlink/?linkid=2185220) in the [SharePoint admin center](https://go.microsoft.com/fwlink/?linkid=2185219), from Sites > Active sites because the certain site templates and URLs are not included included as Active Sites. See [Manage sites in the SharePoint admin center](https://learn.microsoft.com/sharepoint/manage-sites-in-new-admin-center) for more information. 

Reviewer: ```suggestion
Note that you may see differences between these sites and those listed on the [Active sites page](https://go.microsoft.com/fwlink/?linkid=2185220) in the [SharePoint admin center](https://go.microsoft.com/fwlink/?linkid=2185219) because certain site templates and URLs are not included as active sites. See [Manage sites in the SharePoint admin center](https://learn.microsoft.com/sharepoint/manage-sites-in-new-admin-center) for more information. 
```
